tools/buildHelper/buildHelper.psm1

```diff
@@ -92,7 +92,7 @@ function Get-ImageList
 
     if ($Channel -in 'community-stable', 'all')
     {
-        Get-ChildItem -Path $communityStablePath -Directory | Select-Object -ExpandProperty Name | Where-Object { $dockerFileNames -notcontains "clearlinux" } | Write-Output
+        Get-ChildItem -Path $communityStablePath -Directory | Select-Object -ExpandProperty Name | Where-Object { $dockerFileNames -notcontains "clearlinux" } | Where-Object { $dockerFileNames -notcontains "parrot" } | Where-Object { $dockerFileNames -contains "kali" } | Write-Output
```

Comment:
I was fine with this when it was one, but let's make this a property in metadata.json.